.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": true
```

Comment:
Would be nice to add a rule for trailing commas.

Reply:
I will do that ;)

Reply:
Done :)